verifier/README.md

```diff
@@ -6,16 +6,16 @@ While [Miden](../miden) crate also contains verifier functionality, if a project
 ## Usage
 This crate exposes a `verify()` function which can be used to verify proofs of program execution. The function takes the following parameters:
 
-* `program_hash: Digest` - a hash of the program to be verified (represented as a 32-byte digest).
+* `program_info: ProgramInfo` -  a structure containing the hash of the program to be verified (represented as a 32-byte digest), and the hashes of the Kernel procedures used to execute the program.
```

Comment:
nit (question): should `kernel` be lower case?

Reply:
Yes - fixed!